indico/modules/users/controllers.py

```diff
@@ -522,18 +525,83 @@ def _process(self):
         return jsonify_data()
 
 
+class UserSearchResultSchema(mm.ModelSchema):
+    class Meta:
+        model = User
+        fields = ('id', 'identifier', 'email', 'affiliation', 'full_name')
+
+
+search_result_schema = UserSearchResultSchema()
+
+
 class RHUserSearch(RHProtected):
     """Search for users based on given criteria"""
 
-    @use_args({
+    def _serialize_pending_user(self, entry):
+        first_name = entry.data.get('first_name') or ''
```

Comment:
why not `entry.data.get('first_name', '')`?

Reply:
That won't cover the value being `None` in the dict